resources/measures/ResidentialGeometryCreateSingleFamilyDetached/measure.rb

```diff
@@ -266,6 +266,9 @@ def run(model, runner, user_arguments)
     foundation_type = runner.getStringArgumentValue('foundation_type', user_arguments)
     foundation_height = runner.getDoubleArgumentValue('foundation_height', user_arguments)
     attic_type = runner.getStringArgumentValue('attic_type', user_arguments)
+    if attic_type == 'finished attic'
+      num_floors -= 1
+    end
```

Comment:
Same comment here.

Reply:
Interesting, we were already using the number of stories above grade (_including finished attics_) for all of our downstream measures. Seems bizarre that ResStock was using a different convention.

```
    if attic_type == 'finished attic'
      num_floors += 1
    end
    model.getBuilding.setStandardsNumberOfAboveGroundStories(num_floors)
```